### DISCUSSION TITLE
Note able to install Fleet agent in any host machine.

### DISCUSSION THREAD

**Discussion by yonikhil** [Security-Onion-Solutions/securityonion#7521] [Unsupported Versions] [answered]
Hi,

I'm not able to install Fleet agent in host machine. Does security onion, Kibana dashboard support Elastic Agent?

**Comment by dougburks** [comment, marked as answer]
The Elastic Agent currently requires opening up the Elasticsearch port to all of your Elastic Agents. Elastic is currently working on allowing Elastic Agent to send to Logstash instead. We are waiting on that feature since it reduces the attack surface.

In the meantime, you can use another agent in the Elastic family (Winlogbeat or some other Beat):
https://docs.securityonion.net/en/2.3/beats.html
